admin/client/src/components/Tabs/TabItem.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import SilverStripeComponent from 'lib/SilverStripeComponent';
+import { Tab } from 'react-bootstrap-ss';
+
+class TabItem extends SilverStripeComponent {
+  getTabProps() {
+    return {
+      eventKey: this.props.name,
+      title: this.props.title,
+      disabled: this.props.disabled,
+      className: [this.props.extraClass, this.props.className].join(' '),
```

Comment:
We should merge this with other props passed into the component as a standard practice, there's heaps more you can specify via react-bootstrap: https://react-bootstrap.github.io/components.html#tabs. Please implement this on all react-bootstrap wrappers